### PR TITLE
Adding note about Spanish translation

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

File: es/book.tex [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 \begin{document}
 
-\title{Enseñar tecnología en comunidad}
+\title{Enseñando tecnología juntos}
```
Comment (yabellini) [inline es/book.tex:6]: \title{Enseñar tecnología en comunidad}

Comment (gvwilson) [reply es/book.tex]: Fixed.

**Comment by yabellini** [comment]
@gvwilson do you want I merge this and then I make the changes in dedication?

**Comment by gvwilson** [comment]
Si, et danke

**Pull request merged**